components/modal/modal.html

```diff
@@ -0,0 +1,98 @@
+                        class="fab fa-twitter-square"></i></a>
+                <a href="https://www.linkedin.com/in/tushargoyal29/" class="nav-icons anchor-style pointer"><i
+                        class="fab fa-linkedin"></i></a>
+            </div>
+        </nav>
+    </header>
+    <div class="main-with-sidebar">
+        <div class="sidebar-wrapper">
+            <aside class="sidebar">
+                <h1>Get started</h1>
+                <hr class="hr-margin">
+                <ul class="sidebar-list">
+                    <li class="sidebar-items"><a class="anchor-style links "
+                            href="/documentation/documentation.html">Introduction</a></li>
+                    <li class="sidebar-items"><a class="anchor-style links" href="">Color</a></li>
+                    <li class="sidebar-items"><a class="anchor-style links page-active"
+                            href="/components/text-utils/text-utils.html">Typography</a></li>
+                </ul>
+                <h1>Components</h1>
+                <hr class="hr-margin">
+                <ul class="sidebar-list">
+                    <li class="sidebar-items"><a class="anchor-style links"
+                            href="/components/avatar/avatar.html">Avatar</a></li>
+                    <li class="sidebar-items"><a class="anchor-style links"
+                            href="/components/alert/alert.html">Alert</a></li>
+                    <li class="sidebar-items"><a class="anchor-style links"
+                            href="/components/badge/badge.html">Badge</a></li>
+                    <li class="sidebar-items"><a class="anchor-style links"
+                            href="/components/button/button.html">Button</a></li>
+                    <li class="sidebar-items"><a class="anchor-style links" href="/components/cards/card.html">Cards</a>
+                    </li>
+                    <li class="sidebar-items"><a class="anchor-style links"
+                            href="/components/image/image.html">Image</a></li>
+                    <li class="sidebar-items"><a class="anchor-style links"
+                            href="/components/input/input.html">Input</a></li>
+                    <li class="sidebar-items"><a class="anchor-style links" href="/components/list/list.html">Lists</a>
+                    </li>
+                    <li class="sidebar-items"><a class="anchor-style links"
+                            href="/components/navigation/navigation.html">Navigation</a></li>
+                    <li class="sidebar-items"><a class="anchor-style links"
+                            href="/components/modal/modal.html">Modal</a></li>
+                    <li class="sidebar-items"><a class="anchor-style links"
+                            href="/components/ratings/ratings.html">Ratings</a></li>
+                    <li class="sidebar-items"><a class="anchor-style links"
```

Comment:
For all the li elements here the code should be more elegant .

Reply:
use prettier app in vscode for proper formatting.